### PR TITLE
fix(DB/creature): Improve Azuregos' mechanic

### PR DESCRIPTION
**Pull request by Pyromanihack**
## Changes Proposed:
-  Unpin the 128 bit from mechanic_immune_mask
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13594
- Closes https://github.com/chromiecraft/chromiecraft/issues/4315

## Tests Performed:

- [Azuregos test.webm](https://user-images.githubusercontent.com/82369562/198898342-54b3b8f4-7085-4e96-8ba5-60d60af3712d.webm)

- screenshot2

## How to Test the Changes:

1. .learn 6343
2. .go c id 6109
3. See if the Thunderclap debuff affects the Boss

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
Hey, could you use an UPDATE query instead of an INSERT? :)
Keira does it automatically if you choose the Diff-query option

File: data/sql/updates/pending_db_world/Azuregos [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+UPDATE creature_template SET mechanic_immune_mask&~(128) WHERE (entry = 6109);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/Azuregos:2]:
```suggestion
UPDATE `creature_template` SET `mechanic_immune_mask`=`mechanic_immune_mask`&~(128) WHERE (entry = 6109);
```

Comment (Gultask) [reply data/sql/updates/pending_db_world/Azuregos:2]: Also, you need to add `.sql` to the filename.

You can create .sql files automatically as well by using executing the `create_sql.sh` file in your `pending_db_world` folder

**Comment by Pyromanihack** [comment]
> Hey, could you use an UPDATE query instead of an INSERT? :) Keira does it automatically if you choose the Diff-query option

Sure, Just done

**Review by Gultask — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@Gultask please make sure all PRs have always a workflow label so they don't get forgotten.

E.g. is this `[Waiting to be tested]` ? `[To Be Merged]` or what else is it waiting for? :)

In this case, G=given that the author tested it already and he is a trusted user, we can mark it as ready to be merged.

**Comment by FrancescoBorzi** [comment]
@Gultask @Pyromanihack also, for this and other PRs that fix issues coming from ChromieCrat, we need to also add the original issue here so it will be automatically closed when this PR is merged.

For this PR I just did it:

image

please make sure the other PRs have it as well

**Comment by Nyeriah** [comment]
Thanks for the PR 👍

**Pull request merged**